lib/adal.js

```diff
@@ -672,18 +708,18 @@ AuthenticationContext.prototype.saveTokenFromHash = function (requestInfo) {
                 this._saveItem(this.CONSTANTS.STORAGE.SESSION_STATE, requestInfo.parameters[this.CONSTANTS.SESSION_STATE]);
             }
 
-            var keys, resource;
+            var keys;
 
             if (requestInfo.parameters.hasOwnProperty(this.CONSTANTS.ACCESS_TOKEN)) {
                 this.info('Fragment has access token');
-                resource = this._getResourceFromState(requestInfo.stateResponse);
                 if (!this._hasResource(resource)) {
                     keys = this._getItem(this.CONSTANTS.STORAGE.TOKEN_KEYS) || '';
                     this._saveItem(this.CONSTANTS.STORAGE.TOKEN_KEYS, keys + resource + this.CONSTANTS.RESOURCE_DELIMETER);
                 }
                 // save token with related resource
                 this._saveItem(this.CONSTANTS.STORAGE.ACCESS_TOKEN_KEY + resource, requestInfo.parameters[this.CONSTANTS.ACCESS_TOKEN]);
                 this._saveItem(this.CONSTANTS.STORAGE.EXPIRATION_KEY + resource, this._expiresIn(requestInfo.parameters[this.CONSTANTS.EXPIRES_IN]));
+                this.verbose('Token successfully updated in local storage.');
```

Comment:
Can you update this to "access token updated in storage."? The storage used could be session storage.